extensions/blocks/podcast-player/style.scss

```diff
@@ -126,11 +132,23 @@ $player-float-background: $light-gray-200;
 		order: 99; // high number to make it always appear after the audio player
 		padding: 0 $player-grid-spacing;
 		margin-bottom: $player-grid-spacing;
-		color: $dark-gray-500;
 		font-size: $description-font-size;
 		line-height: 1.6;
 	}
 
+	// Apply `secondary` color to the podcast title.
+	.jetpack-podcast-player__container:not(.has-secondary) { // no custom -> default color.
+		.jetpack-podcast-player__track-description {
+			color: $dark-gray-500;
+		}
+	}
```

Comment:
I think we can reduce the specificity on this one too to just 
```
.jetpack-podcast-player__track-description {
	color: $dark-gray-500;
}
```

Reply:
I don't think we need the `.jetpack-podcast-player__container:not(.has-secondary) {` part